k9mail/src/main/java/com/fsck/k9/fragment/ICursorContextMenuSupplier.java

```diff
@@ -0,0 +1,9 @@
+package com.fsck.k9.fragment;
+import android.database.Cursor;
+import android.view.ContextMenu;
+/**
+ Created by Kamil Rajtar on 05.11.17. */
```

Comment:
Remove this and similar such docs - we attribute via changelogs and Git commits, not in the code.